### PR TITLE
Remove unused imports after manual review

### PR DESCRIPTION
**Pull request by bw4sz**
Following up from https://github.com/pytorch/vision/pull/3226#issuecomment-756055585. @datumbox 

This was my first week with pytorch coming from a few years of tensorflow. I try to contribute in any small way to get a feeling for the codebase. I went through and looked for unused imports using my IDE (WingPro 7) which identifies unused imports. I then searched the script to confirm, followed by a manual inspection. I intentionally used a very light hand, there were many situations where an unused import was in a commented out line, or in the 'examples' in the docstring. I choose leave those in, since this was just a small style update. In a roundabout way this is also a nice test of code coverage and unit tests to make sure pull requests are adequately evaluated. I hope it is useful. 

Thanks for your work.

### CODE REVIEW

File: torchvision/datasets/voc.py
Comment (datumbox) [inline torchvision/datasets/voc.py:7]: I believe List is not used either?
```suggestion
from typing import Any, Callable, Dict, Optional, Tuple
```

File: torchvision/models/detection/mask_rcnn.py
Comment (datumbox) [inline torchvision/models/detection/mask_rcnn.py:4]: I believe that the following two import are also not needed?
`import torch.nn.functional as F` 
`from torchvision.ops import misc as misc_nn_ops`

**Review by datumbox — COMMENTED** [review]
@bw4sz Welcome to PyTorch and thanks for your contribution.

Overall the changes look great. I left a couple of comments, let me know what you think.

**Review by bw4sz — COMMENTED** [review]
Yes, some of these I saw and left, others I missed.

**Comment by datumbox** [comment]
> Yes, some of these I saw and left, others I missed.

You can safely remove the 2 remaining ones I'm flagging on the unresolved comment. Once this is fixed we can merge the PR. :)

**Review by datumbox — APPROVED** [review]
LGTM! 

Thanks for the contribution @bw4sz. Looking forward to more PRs on the future. :)

**Comment by datumbox** [comment]
@bw4sz Hmm. Unfortunately I can not merge because the branch is out of date with the master branch and I don't have permissions to modify your branch. 

Either merge with the latest master or click the `Allow edits and access to secrets by maintainers` checkbox on the right.

**Pull request merged**
